lib/roast/dsl/cogs/cmd.rb

```diff
@@ -18,17 +29,20 @@ def initialize(output, error, status)
         end
 
         class Config < Cog::Config
+          #: () -> void
           def print_all!
             @values[:print_all] = true
           end
 
+          #: () -> bool
           def print_all?
             !!@values[:print_all]
           end
         end
 
         use_config_class Config
 
+        #: () -> void
```

Comment:
This is not a void return.
```suggestion
        #: () -> Output
```